frontend/src/component/common/FormTemplate/FormTemplate.tsx

```diff
@@ -36,7 +36,6 @@ const StyledContainer = styled('section', {
     width: '100%',
     display: 'flex',
     margin: '0 auto',
-    overflow: 'hidden',
```

Comment:
Having this set broke `position: sticky` in child components. Removing this had no noticeable negative side-effects that I could find 🤷 

Seems like an old setting: 
https://github.com/Unleash/unleash-frontend/pull/739/files#diff-548c3ebed76c71107c101d96f139fb5fb32649ddd404a87fa1d87b12bf2254b5R10